src/bloaty.cc

```diff
@@ -1637,6 +1637,8 @@ bool BloatyMain(int argc, char* argv[], const InputFileFactory& file_factory,
       verbose_level = 2;
     } else if (strcmp(argv[i], "-vvv") == 0) {
       verbose_level = 3;
+    } else if (strcmp(argv[i], "-w") == 0) {
+      max_label_len = 100000000;
```

Comment:
Why not std::numeric_limits<size_t>::max() or SIZE_MAX or however your lowest denominator platform spells it?
((size_t)-1?)

Also, mention in help/usage string?

Otherwise LGTM.